config/environments/production.rb

```diff
@@ -21,6 +21,8 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  config.cache_store = :redis_cache_store, { url: ENV.fetch('REDIS_CACHE_URL') }
```

Comment:
Having a look through https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-rediscachestore, it seems there could be some further settings we could tweak like `reconnect_attempts` and possibly setting an `error_handler` etc. the other defaults seem ok for now though

Reply:
I reckon we should also have a fallback if the cache URL isn't defined

Reply:
I guess we'd have to fall back to in-memory

Reply:
I wanted to check for the presence of the `REDIS_CACHE_URL` in production, hence `fetch` instead of `[]`,

Reply:
I think I agree with Vignesh — it's better to be bold and rely on the cache being there, then fail noisily if not